### ISSUE TITLE
Some fields missing

### ISSUE THREAD

**Issue by antagomir** [fennicahub/fennica#53]
The Rahti site http://fennica-fennica.rahtiapp.fi/ seems to be missing some fields that we have previously harmonized, notably e.g. page count and physical dimension info, perhaps also other. Compare:
https://github.com/comhis/fennica/blob/master/inst/examples/overview.md

Todo:

1) List what fields are missing compared to the earlier version

2) Decide which to bring in and which to discard

3) Bring in the omitted ones.

**Comment by antagomir** [comment]
Ok the missing ones seem to be listed here https://github.com/fennicahub/fennica/blob/master/inst/examples/main_in_transfer.R

**Comment by JuliaMatveeva217** [comment]
Yes, they are missing. 

Has somebody worked on them? Do we need to create polish_functions for these fields from scratch?

**Comment by antagomir** [comment]
No. The earlier polish functions are available. They just need to be separated from that script, to have them done per field.

**Comment by JuliaMatveeva217** [comment]
Ok, I am working on it
